XYZDrivers/src/main/webapp/index.html

```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Start Page</title>
+        <title>Home Page</title>
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <link rel="stylesheet" type="text/css" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
     </head>
     <body>
-        <h1>Hello World!</h1>
+        <nav class="navbar navbar-default">
+            <div class="container-fluid">
+              <div class="navbar-header">
+                <a class="navbar-brand" href="#">XYZ Drivers Association</a>
+              </div>
```

Comment:
Was going to suggest instead referencing the navbar from a separate html page using JQuery, but if we're not supposed to use JavaScript right?